### DISCUSSION TITLE
How to collect random sample of tweets for a given location and time?

### DISCUSSION THREAD

**Discussion by vijetadeshpande** [DocNow/twarc#655] [Q&A] [answered]
I am using the search command with a specified location (long, lat, and radius), time (1st Jan 2019 to 31st Dec 2019), and a limit of 1000. For the search command, I am not using any keywords in the query. With this setup, I am expecting to collect a pool of general tweets but what I don't know is,

1. How those 1000 tweets will be selected? From newest to oldest i.e. starting from 31st Dec and going till 1st Jan?
2. Ideally I want a pool of uniformly sampled tweets for a given place and period. Would the above setup be equivalent to uniform sampling?
3. Is there any other way of random/uniform sampling?

**Comment by SamHames** [comment, marked as answer]
To answer your questions:
1. The tweets will be reverse chronological, with a limit of 1000 you'll get at least 1000 tweets, starting with the newest first.
2. No, this would not be the same as uniform sampling.
3. There's no support in the API for any kind of sampling: you could only get close to a true random sample by collecting everything and then sampling as a post-processing step. There might be ways to take samples at random times throughout that window, but they probably wouldn't meet the requirements for a random sample.

**Reply by vijetadeshpande** [reply]
This is super helpful, thanks, Sam!
